src/taipy/gui/config.py

```diff
@@ -170,7 +172,7 @@ def _handle_argparse(self, args):  # pragma: no cover
             config["use_reloader"] = False
         if args.ngrok_token:
             config["ngrok_token"] = args.ngrok_token
-        if args.force_https:
+        if args.use_https:
             config["force_https"] = True
```

Comment:
Should we change the config param from force_https to use_https everywhere for the sake of uniformity and doc ?